scene/gui/gradient_edit.cpp

```diff
@@ -126,8 +132,15 @@ void GradientEdit::_gui_input(const Ref<InputEvent> &p_event) {
 		}
 	}
 
-	//Hold alt key to duplicate selected color
-	if (mb.is_valid() && mb->get_button_index() == 1 && mb->is_pressed() && mb->get_alt()) {
+	// Show context menu on right click.
+	if (mb.is_valid() && mb->get_button_index() == BUTTON_RIGHT && mb->is_pressed()) {
+		context_menu->set_position(get_global_transform().xform(mb->get_position()));
```

Comment:
Unfortuntely, the context menu doesn't spawn at the right position. I don't know why, as I used the same method in https://github.com/godotengine/godot/pull/38593 and it works there.